.agents/skills/cherry-pick-pr/scripts/cherry_pick_pr.py

```
@@ -0,0 +1,834 @@
+        if stdout:
+            print(stdout)
+        if stderr:
+            print(stderr, file=sys.stderr)
+        if proc.returncode == 0:
+            state["cherry_pick_status"] = "applied"
+            state["test_status"] = "not_run"
+            save_state(repo_root, state)
+            emit(
+                args.output,
+                {**payload, "status": "applied"},
+                lines + ["Cherry-pick applied cleanly"],
```

Ruff (0.15.7)

[warning] Consider `[*lines, "Cherry-pick applied cleanly"]` instead of concatenation

Replace with `[*lines, "Cherry-pick applied cleanly"]`

(RUF005)